app/edit-ui/src/util/classnames.js

```diff
@@ -0,0 +1,27 @@
+export default function classnames(...args) {
```

Comment:
It would be useful to have some comments explaining what this util is for and what it does exactly. Is it meant to live in tandem with or replace the existing `Style` class, and why?

Reply:
Oh, this is just a copypaste of classnames – https://www.npmjs.com/package/classnames (will add a comment to that effect).

I use it in the component library, because it's the standard way to do it (10k+ dependents on npm, 3M+ weekly downloads). It's so tiny and the API so stable that I didn't feel like installing it as a dependency, so I copied it instead (I rewrote it to ES6 code, but otherwise it's the same).

In Publish, I'd continue using Style just so that we don't introduce inconsistent ways of doing things. In new repos, I'd use classnames.